sympy/functions/elementary/trigonometric.py

```diff
@@ -717,6 +717,7 @@ def _fermatCoords(n):
             # if n can be factored in terms of Fermat primes with
             # multiplicity of each being 1, return those primes, else
             # False
+            from sympy import chebyshevt
```

Comment:
That's just moving a bottom import into a function body.
The best way to test that is via code coverage; I'm not aware of how this is organized in SymPy.

The motive for doing that was to avoid an import cycle. I'm not sure whether it's still there (the cycle happened during an intermediate state when the main change wasn't bug-free yet), but then I think that if some cycle happens accidentally, that's a good reason to move it to a place where it can't create a cycle. The other reason is that bottom imports make maintenance slightly harder (maintainers have now two places to check for an import).
In the end, I decided to leave this in.

I'll remove it if you find that inacceptable anyway.